benchmarks/user/benchmark.py

```diff
@@ -41,10 +43,43 @@
         'run': acoustic_run,
         'setup': acoustic_setup,
         'default-section': 'section0'
+    },
+    'acoustic_ssa': {
+        'run': acoustic_ssa_run,
+        'setup': acoustic_ssa_setup,
+        'default-section': 'section0'
     }
 }
 
 
+def run_op(solver, operator, autotune, **options):
+    """
+    Initialize any necessary input and run the operator associated with the solver.
+    """
+    # Get the operator if exist
+    try:
+        op = getattr(solver, operator)
+    except AttributeError:
+        raise AttributeError("Operator %s not implemented for %s" % (operator, solver))
+
+    # THis is a bit ugly but not sure how to make clean input creation for different op
```

Comment:
@FabioLuporini  this is not very pretty at all, but each operator have different mandatory inputs (trying to avoid computing zeros for benchmark) so this do the job for now.